### DISCUSSION TITLE
Multi datacenter strategy

### DISCUSSION THREAD

**Discussion by werenzo** [NetApp/harvest#1141] [Q&A] [answered]
Hello,

In a multi datacenter environment, do you suggest to distribute pollers across datacenter, or let a single harvest instance monitor all the Ontap cluster from a single dc?

Thanks a lot

Lorenzo

**Comment by cgrinds** [comment, marked as answer]
hi @werenzo both work well and are used by many. Most people end up using a single harvest instance since it's easier to manage while still scaling to high node counts.

As @rahulguptajss mentioned on Slack, we have customers monitoring 300+ nodes on a single instance of Harvest running on a machine with 24 GB of memory.
